R/shiny.R

```diff
@@ -134,16 +134,21 @@ run <- function(file = "index.Rmd", dir = dirname(file), default_file = NULL,
   yaml_front <- if (length(target_file)) yaml_front_matter(target_file)
   runtime <- yaml_front$runtime
   theme <- render_args$output_options$theme
```

Comment:
@cpsievert as I am going through shiny prerendered code base to understand better how it works, I am just wondering what this line is used for ? It does not seem to be used now with the PR changes. 

Do you recall the aim of catching a special handling when `theme` would be passed inside `rmarkdown::run(renderArgs=)` ?